xcell/mappers/mapper_DESY1gc.py

```diff
@@ -38,23 +43,38 @@ def __init__(self, config):
         self.nl_coupled = None
 
     def get_catalog(self):
+        """
+        Returns the mapper's binned catalog.
```

Comment:
Return the catalog of the requested redshift bin

Reply:
BTW, In the documentation one writes "Return" instead of "Returns", as in an order. https://peps.python.org/pep-0257/#one-line-docstrings